SchlafEin.m

```diff
@@ -1,7 +1,7 @@
 % SchlafEin.m 
 % Simple help for sleep scoring
-% 16.09.2021 Version 0.946beta Steffen Gais
-% 0.946 - initial version for GitHub
+% 16.09.2021 Steffen Gais
+% version 0.947 - initial version for GitHub

```

Comment:
The update script will find any line that is a comment (`% ...`) and contains the word "version". Then it will look for the last floating number in that line.